src/snowflake/cli/api/utils/rendering.py

```diff
@@ -105,17 +72,21 @@ def generic_render_template(
     Returns:
         None
     """
-    env = jinja2.Environment(
-        loader=jinja2.loaders.FileSystemLoader(template_path.parent),
-        keep_trailing_newline=True,
-        undefined=jinja2.StrictUndefined,
+    env = _env_bootstrap(
+        Environment(
+            loader=loaders.FileSystemLoader(template_path.parent),
+            keep_trailing_newline=True,
+            undefined=StrictUndefined,
+        )
```

Comment:
This is being used in `snow app init`, which currently uses the regular jinja syntax `{{}}`. I think we can constrain it such that all client side rendering must have `&{}` only. I can change our templates to conform to that. WDYT?

Reply:
I don't agree, for example imagine creating a project for template that includes client side templating...

Reply:
Native app has custom client-side template logic now that uses the standard jinja2 syntax. We should likely also support the Snowflake CLI syntax, but we can't remove support for the current behaviour without risking breaking existing users. I do agree with Tomasz that the Snowflake CLI does make it easier for clients to mix their own client-side template with ours.